nixtla/nixtla_client.py

```diff
@@ -629,6 +629,15 @@ def _make_partitioned_requests(
             resp["weights_x"] = None
         else:
             resp["weights_x"] = [res["weights_x"] for res in results]
+        if (
+            "feature_contributions" not in first_res
+            or first_res["feature_contributions"] is None
+        ):
```

Comment:
The field will always be there, we just have to check if its none or not.
```suggestion
        if first_res["feature_contributions"] is None:
```